testsuite/src/main/resources/csrf/methods/views/csrf-ok-post.jsp

```diff
@@ -0,0 +1,18 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<!doctype html>
+<html>
+<head>
+    <title>CSRF Custom Exception Test</title>
+</head>
+<body>
+<h1>CSRF Custom Exception Test</h1>
+<form id="form" action="${mvc.uri('doPost')}" method="post" accept-charset="utf-8">
+
+    <input type="hidden" name="_method" value="DELETE">
+
```

Comment:
Here is Delete ?

Reply:
Oh yes, good catch, thanks! Seems I forgot to change it in this form and the PATCH form.

Reply:
Should be fixed now :+1: